index.html

```diff
@@ -2183,8 +2146,7 @@ <h3>Presentations</h3>
   ],
   "id": "urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5",
   "type": ["VerifiablePresentation", "ExamplePresentation"],
-  <span class="highlight">"verifiableCredential": [{ <span class="comment">...</span> }],
-  "proof": [{ <span class="comment">...</span> }]</span>
```

Comment:
removed empty proof from example

Reply:
Then convert this into a real example that shows a minimal proof using both securing mechanisms. We don't gain anything by NOT showing people what a secured a VC or VP looks like.